docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/Regression/GamAdvanced.cs

```diff
@@ -0,0 +1,155 @@
+﻿using System;
+using System.Collections.Generic;
+using Microsoft.ML;
+using Microsoft.ML.Data;
+
+namespace Samples.Dynamic.Trainers.BinaryClassification
+{
+    public static class GamAdvanced
+    {
+        // This example requires installation of additional NuGet package
+        // <a href="https://www.nuget.org/packages/Microsoft.ML.FastTree/">Microsoft.ML.FastTree</a>.
```

Comment:
in pother places it's on top of class.
Also I don't see tt file for it.
Is it something you forget to remove from PR, or you actually intent to commit it?

Reply:
This sample is orthogonal to those added using tt. It is more like a tutorial. I didn't include it as API documents but users can still find it on Github.

---
In reply to: [275455001](https://github.com/dotnet/machinelearning/pull/3319#discussion_r275455001) [](ancestors = 275455001)